src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import usersFromServer from '../../api/users';
+import { InputValues } from '../../types/inputValues';
+
+type Props = {
+  onAddTodo: ({ title, user }: InputValues) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onAddTodo }) => {
+  const [user, setUser] = useState(0);
+  const [title, setTitle] = useState('');
+  const [titileError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  useEffect(() => {
+    let timer: ReturnType<typeof setTimeout>;
+
+    if (titileError || userError) {
+      timer = setTimeout(() => {
+        setTitleError(false);
+        setUserError(false);
+      }, 1000);
+    }
+
+    return () => {
+      clearTimeout(timer);
+    };
+  }, [titileError, userError]);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setTitleError(!title);
+    setUserError(!user);
+
+    if (!user || !title) {
+      return;
+    }
+
+    onAddTodo({ title, user });
+
+    setUserError(false);
+    setTitleError(false);
+    setTitle('');
+    setUser(0);
```

Comment:
Let's create a separate function for resetting

Reply:
Sorry, the first time only part of the comments were downloaded( 
I am changing this now. 
But I was sure that it is ok to use it like this if we use it only once)